### DISCUSSION TITLE
Is it possible to create a checkbox group in strapi?

### DISCUSSION THREAD

**Discussion by marshallshelly** [strapi/strapi#6580] [Legacy Topics]
I wanted to have a group of checkbox, where new checkboxes can be added while adding content to the content type.

Is it possible?

**Update**
I tried creating a component with boolean's. Is this the ideal way? If not, please advise me

**Comment by Bjorn-Eric-Abr** [comment]
I was looking for this myself - is the boolean component thing our best solution?

**Comment by derrickmehaffy** [comment]
Currently we don't have a multi-select checkbox option, either booleans or using something like the `manyWay` relation type (far right in the relational matrix) is the best option for the moment.
